consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.moj.sdt.cmc.consumers;
+
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import uk.gov.moj.sdt.cmc.consumers.api.IBreathingSpace;
+import uk.gov.moj.sdt.cmc.consumers.converter.XmlToObjectConverter;
+import uk.gov.moj.sdt.cmc.consumers.exception.CMCException;
+import uk.gov.moj.sdt.cmc.consumers.model.breathingspace.BreathingSpaceRequest;
+import uk.gov.moj.sdt.consumers.api.IConsumerGateway;
+import uk.gov.moj.sdt.consumers.exception.OutageException;
+import uk.gov.moj.sdt.consumers.exception.TimeoutException;
+import uk.gov.moj.sdt.domain.api.IIndividualRequest;
+import uk.gov.moj.sdt.domain.api.ISubmitQueryRequest;
+
+@Component("CMCConsumerGateway")
+public class CMCConsumerGateway implements IConsumerGateway {
```

Comment:
Bit of inconsistency in the case used for acronyms.  CMC is capitalised, e.g. CMCConsumerGateway, but API in CMCApi is in camel-case.  Likewise, XML in XmlToolConverter is in camel-case.  Should there be a consistent style?

Reply:
renamed CmcApi already to CMCApi